Makefile

```diff
@@ -212,18 +212,31 @@ bios-flash: $(BIOS_FILE) bios-flash-$(PLATFORM)
 # TFTP booting stuff
 # --------------------------------------
 # TFTP server for minisoc to load firmware from
+ATFTPD  := $(shell which atftpd 2>/dev/null)
+INTFTPD := $(shell which in.tftpd 2>/dev/null)
```

Comment:
These will run every time the Makefile is opened which can make things slow. Could we instead do the if/then/else in shell inside the targets?

Reply:
Ah, right.  I'll change it.